stigaview_static/main.py

```
@@ -75,31 +105,44 @@ def process_product(
+    total_files = _get_total_files(input_path, products)
+    with tqdm(total=total_files, desc="Processing all STIG files", unit="file") as pbar:
+        for product in products:
+            product_path = pathlib.Path(input_path) / product.short_name
+            if not product_path.exists():
+                logging.error(
+                    f"Unable to find path for {product.short_name} at {product_path}"
+                )
+                exit(4)
+            product, srgs = process_product(product, product_path, pbar)
+            for srg, controls in srgs.items():
+                if srg not in srgs_dict.keys():
```

Ruff (0.11.9)

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)